integrations/jira/.port/resources/port-app-config.yaml

```diff
@@ -47,15 +47,15 @@ resources:
             status: .fields.status.name
             issueType: .fields.issuetype.name
             components: .fields.components
-            creator: .fields.creator.emailAddress
+            creator: .fields.creator.emailAddress // ""
             priority: .fields.priority.name
             labels: .fields.labels
             created: .fields.created
             updated: .fields.updated
-            resolutionDate: .fields.resolutiondate
+            resolutionDate: .fields.resolutiondate // ""
           relations:
             project: .fields.project.key
-            parentIssue: .fields.parent.key
+            parentIssue: .fields.parent.key // ""
             subtasks: .fields.subtasks | map(.key)
-            assignee: .fields.assignee.accountId
+            assignee: .fields.assignee.accountId // ""
```

Comment:
why?

Reply:
When running the integration, I noticed certain logs showing that some fields returned `null` or were missing. I added the `// ""` part to explicitly handle null values.

Reply:
is that an issue with us relying on a key that doesn't actually exists? or is available in specific cases?

Reply:
do we document is somewhere when this data is available?

Reply:
No, we aren't really relying on it per se. It is the assignee field which may not be present if an issue hasn't been assigned to anyone yet. It may be null in such cases. It doesn't need to be documented since it is common knowledge, else we'll have to document all the fields including an absent description.

Reply:
The user themselves are the ones who control that field. It isn't on us.